scan/src/bindings/Bech32.re

```diff
@@ -4,6 +4,26 @@ type decoded_t = {
   words: array(int),
 };
 
+let safeDecode: string => (decoded_t, bool) = [%bs.raw
```

Comment:
Good try. But I don't think you need to go bs_raw here. You can catch exception in ReasonML. https://reasonml.github.io/docs/en/exception.